sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_providers/local_provider.py

```diff
@@ -60,16 +60,17 @@ def _raise_if_unsupported(self, operation, algorithm):
         if operation not in self._allowed_ops:
             raise AzureError('This key does not allow the "{}" operation'.format(operation))
 
-    def encrypt(self, algorithm, plaintext):
-        # type: (EncryptionAlgorithm, bytes) -> EncryptResult
+    def encrypt(self, algorithm, plaintext, **kwargs):
+        # type: (EncryptionAlgorithm, bytes, **Any) -> EncryptResult
         self._raise_if_unsupported(KeyOperation.encrypt, algorithm)
-        ciphertext = self._internal_key.encrypt(plaintext, algorithm=algorithm.value)
-        return EncryptResult(key_id=self._key.kid, algorithm=algorithm, ciphertext=ciphertext)
+        iv = kwargs.pop("iv", None)
```

Comment:
Good point! I had decided against doing so since it's only necessary for some symmetric algorithms, but after a second look it does make sense to make it an optional parameter.